laser_encoders/__init__.py

```diff
@@ -12,4 +12,8 @@
 #
 # -------------------------------------------------------
 
-from laser_encoders.download_models import initialize_encoder, initialize_tokenizer
+from laser_encoders.download_models import (
```

Comment:
While we're here, can we move these out of `download_models.py` into more relevant files? For example: move `inititialize_tokenizer` into `laser_tokenizer.py`.